beeline/pom.xml

```diff
@@ -193,6 +193,11 @@
       <version>${powermock.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
@zratkai I see this dependency is again introduced in your last commit, did something change? do we need this? or this was just a commit miss?